lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -390,6 +389,7 @@ DefConstructorI(T_CS('\end{document}'), undef, sub {
     #      push(@boxes, Digest(Tokens(@$ops))); }
     # NOTE: Don't flush gullet (tempting);
```

Comment:
Could you remove the comment above that says `Don't flush gullet (tempting);` ? Seems outdated now :>